docs/src/components/code-block/examples.tsx

```diff
@@ -30,25 +30,23 @@ export interface ExampleData {
 
 interface ExamplesSwitcherProps extends HTMLAttributes<HTMLDivElement> {
   examples: ExampleData[]
-  language: Language
-  startExpanded: boolean
 }
 export const ExamplesSwitcher: React.FC<ExamplesSwitcherProps> = props => {
-  const { examples, language, className, startExpanded, ...rest } = props
-
-  const [selectedOption, setSelectedOption] = React.useState(examples[0].value)
+  const [selectedOption, setSelectedOption] = React.useState(
+    props.examples[0].value
+  )
```

Comment:
```suggestion
  const [selectedOption, setSelectedOption] = useState(
    props.examples[0].value
  )
```

nitpick...just because we already import it and use it this way